work_items/kyc-vcs/schemas/kyc-credential.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "KYC Credential Schema",
+  "description": "Schema for a KYC Credential",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "object",
+      "properties": {
+        "familyName": {
+          "type": "string"
+        },
+        "givenName": {
+          "type": "string"
+        },
+        "additionalName": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "birthDate": {
+      "type": "string"
+    },
+    "postalAddress": {
+      "type": "object",
+      "properties": {
+        "addressCountry": {
+          "type": "string"
+        },
+        "addressLocality": {
+          "type": "string"
+        },
+        "addressRegion": {
+          "type": "string"
+        },
+        "postOfficeBoxNumber": {
```

Comment:
Believe we're aiming to use postalCode here, rather than postOfficeBoxNumber?

https://schema.org/postalCode

Reply:
i was wondering if this is a distinct field (PO Box vs zip code)

Reply:
https://github.com/TBD54566975/credentials-working-group/pull/10